packages/twenty-server/src/modules/calendar/repositories/calendar-event.repository.ts

```diff
@@ -97,8 +117,8 @@ export class CalendarEventRepository {
           iCalUID: string;
         }[]
       | undefined = await this.workspaceDataSourceService.executeRawQuery(
-      `SELECT id, "iCalUID" FROM ${dataSourceSchema}."calendarEvent" WHERE "id" = ANY($1)`,
-      [calendarEventIds],
+      `SELECT id, "iCalUID" FROM ${dataSourceSchema}."calendarEvent" WHERE "iCalUID" = ANY($1)`,
+      [iCalUIDs],
```

Comment:
LGTM but keep in mind that we can't make sure this is unique from the DB as of today 👍 (compared to calendarEventIds)